### ISSUE TITLE
"swipeEnabled" and "animationEnabled" don't work properly in createBottomTabNavigator(2.0)

### ISSUE THREAD

**Issue by EternalChildren** [react-navigation/react-navigation#4146]
### Current Behavior
```
...
const TabScreen = createBottomTabNavigator({
  Home: Home,
  Info: Info
}, {
  initialRouteName: 'Home',
  swipeEnabled: true,
  animationEnabled: true,
  tabBarOptions: {
    activeBackgroundColor: '#fff',
    inactiveBackgroundColor: '#fff',
    activeTintColor: '#405BDB',
    inactiveTintColor: '#9B9B9B'
  },
```
swipeEnabled and animationEnabled can't work in ios

### Expected Behavior
Between views can be switched by sliding

### How to reproduce
create a tabnavigatator and set swipeEnabled to true

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | 2.0.0-rc.9
| react-native     | 0.55.3
| node             |  8.10.0
| npm or yarn      | 6.0.0

**Comment by domialex** [comment]
So how do we make a bottom tab navigation with swipe?
Do we just lose a feature?

**Comment by tiagoalves** [comment]
I'm also curios to know whether `createBottomTabNavigator()` will support `animationEnabled`. For now I had to revert to using the deprecated `createTabNavigator()`.

**Comment by EternalChildren** [comment]
I have to say the version 2.0.0-rc9 is not a good choice and the speed of page switching is much slower than v1.5.11. I have revert to using the v1.5.11 .

**Comment by cameronk** [comment]
@brentvatne Curious if this will be added back to `createBottomTabNavigator`, or if the feature is being dropped altogether.

**Comment by SaraChicaD** [comment]
So is the solution to use the _deprecated_ `createTabNavigator` or to wait until it's released, or what? Thank you!

**Comment by brentvatne** [comment]
we won't add this back, but you're welcome to take the deprecated navigator and fork it and create your own project from it which supports these behaviors!

**Comment by domialex** [comment]
Is there an in detail explanation? Just curious.

**Comment by macrozone** [comment]
nice screen transitions are an improvement for react-native over html-based hybrid apps, where this is harder to achieve.

its very irritating that react-navigation introduces this regression. Also the documentation is unclear: https://reactnavigation.org/docs/en/tab-navigator.html

it clearly states, that the import is deprecated and we should just use createBottomTabNavigator. But that's not true.

**Comment by abhi06991** [comment]
This was such an awesome feature. Shame it got removed. We want that back guys!!!!

**Comment by jpamarohorta** [comment]
Did anyone was able to fork `createBottomTabNavigator` to support animations?

**Comment by henninghall** [comment]
Im curious about the reason to remove it. Will we face a big issue if we try to implement it ourselves or has it just not been prioritised for react navigation v2 yet? In other words, if someone made a PR with this feature, would it be accepted or is there any other reason not to include it in the project?

**Comment by aprilmintacpineda** [comment]
If you guys want a bottom navigation with swipe and animation, you can still use the `createMaterialTopTabNavigator` and simply set the `tabBarPosition` to `bottom`.

https://reactnavigation.org/docs/en/material-top-tab-navigator.html#docsNav

**Comment by brentvatne** [comment]
we removed it because this isn't a commonly used feature and the maintenance burden is high. @aprilmintacpineda's approach works too

**Comment by feliperaul** [comment]
@brentvatne Considering the huge number of +1s in this issue and the number of votes in https://react-navigation.canny.io/feature-requests/p/bring-back-animation-for-tabnavigator, could you guys please reconsider? This is really ruining the experience. 

Also, the official 2.0 docs still state that `animationEnabled` and `swipeEnabled` are valid options, but there is no information there that they achieve nothing as it is.

**Comment by Biplovkumar** [comment]
deprecated createTabNavigator() is better than createMaterialTopTabNavigator and createBottomTabNavigator.
So use createTabNavigator.

**Comment by jpamarohorta** [comment]
Just to leave an update opinion.

I’ve talked with several designers and they pointed out that most big apps with bottom tab navigators don’t have animations anymore (with the exception of the Facebook app). So this will not be an issue sooner or later.

**Comment by AndrewHenderson** [comment]
@aprilmintacpineda Is right. [This works great](https://github.com/react-navigation/react-navigation/issues/4146#issuecomment-414049742). Why is no one acknowledging this comment?

**Note: The only deviation is that the tabs always animates.** That was strange anyway. `createBottomTabNavigator` would animate if the user swiped, but jump if the user tapped. This approach provides a consistent UX. 👍

**Comment by Momazo7u7** [comment]
> Just to leave an updated opinion.
> 
> I’ve talked with several designers and they pointed out that most big apps with bottom tab navigators don’t have animations anymore (with the exception of the Facebook app). So this will not be an issue sooner or later.

Agree. The whole swipe to change tab when tabs are in bottom is kinda unnatural, i guess, it works when tabs are on top because the distance to press the tab icon is away, but in bottom tab is as closely as is gonna be.

**Comment by emachta** [comment]
> If you guys want a bottom navigation with swipe and animation, you can still use the `createMaterialTopTabNavigator` and simply set the `tabBarPosition` to `bottom`.
> 
> https://reactnavigation.org/docs/en/material-top-tab-navigator.html#docsNav

This saved my life. Thank you!

**Comment by bohehe** [comment]
> If you guys want a bottom navigation with swipe and animation, you can still use the `createMaterialTopTabNavigator` and simply set the `tabBarPosition` to `bottom`.
> 
> https://reactnavigation.org/docs/en/material-top-tab-navigator.html#docsNav

Great! you saved my life. Thank you ~

**Comment by guilhermeDTNA** [comment]
> If you guys want a bottom navigation with swipe and animation, you can still use the `createMaterialTopTabNavigator` and simply set the `tabBarPosition` to `bottom`.
> 
> https://reactnavigation.org/docs/en/material-top-tab-navigator.html#docsNav

Thanks for the support, my friend! You have saved my work.
